query/apigateway/apigateway_rest_api_stages_configured_with_ssl_certificate.sql

```diff
@@ -0,0 +1,16 @@
+select
+  -- Required Columns
+  'arn:' || partition || ':apigateway:' || region || '::/apis/' || rest_api_id || '/stages/' as resource,
+  case
+    when client_certificate_id is null then 'alarm'
+    else 'ok'
+  end as status,
+  case
+    when client_certificate_id is null then title || ' not configured to use SSL certificates.'
```

Comment:
```suggestion
    when client_certificate_id is null then title || ' uses SSL certificate.'
```

Can a stage use 1 SSL certificate, or multiple? If multiple, please change to ` use SSL certificates.`